doc/decisions/relative.md

```diff
@@ -4,8 +4,8 @@
 
 There is a different behaviour of various plugins whether their name is
 absolute or relative, including:
-1.) mounting the same file somewhere else does not work
-2.) importing somewhere else (other than from where it was exported) does not work
+1. mounting the same file somewhere else does not work
```

Comment:
Its not a new issue you introduce, but now that I see it: I think according to markdown there must be a newline before enumerations.